src/main.cpp

```diff
@@ -82,7 +82,14 @@ void executeBinary(const std::string& binaryFilename
     int exitCode;
 #ifdef USE_MPI
     if (Global::config().get("engine") == "mpi") {
-        exitCode = system(("mpiexec -n " + std::to_string(numberOfProcesses) + " " + binaryFilename).c_str());
+        std::stringstream ss;
```

Comment:
Could you change this so the mpi option exists even if mpi is disabled? i.e. give an error message, or a warning, if mpi is disabled and the option is used.

Reply:
This has been addressed in a follow up at #694.